libs/authorizenet/testing/src/drivers/in-memory/authorize-net.service.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { DaffAuthorizeNetTokenRequest } from '@daffodil/authorizenet';
+
+import { DaffInMemoryAuthorizeNetService } from './authorize-net.service';
+
+describe('Driver | In Memory | AuthorizeNet | AuthorizeNetService', () => {
+  let authorizeNetService: DaffInMemoryAuthorizeNetService;
+	let httpMock: HttpTestingController;
+	const flushedResponse = {
+		response: 'response'
+	};
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientTestingModule
+      ],
+      providers: [
+        DaffInMemoryAuthorizeNetService
+      ]
+    });
+
+    httpMock = TestBed.get(HttpTestingController);
+    authorizeNetService = TestBed.get(DaffInMemoryAuthorizeNetService);
+  });
+
+  afterEach(() => {
+    httpMock.verify();
+  });
+
+  it('should be created', () => {
+    expect(authorizeNetService).toBeTruthy();
+  });
+
+  describe('generateToken', () => {
+    let productId;
+    let qty;
+
+    beforeEach(() => {
+      productId = 'productId';
+      qty = 1;
+    });
+
+    describe('a successful generateToken request', () => {
+      it('should send a post request to `api/authorizenet/generateToken` and respond with a cart', done => {
```

Comment:
`respond with a cart` that doesn't sound right.

Reply:
fixed